scripts/dispatch-dbt-workflow.sh

```diff
@@ -1,7 +1,17 @@
 #!/bin/bash
 
-GH_API_REPO="https://api.github.com/repos/ccao-data/data-architecture"
-GH_DBT_WORKFLOW="test_dbt_models.yaml"
+GH_API_REPOS=(
+    "https://api.github.com/repos/ccao-data/data-architecture"
+    "https://api.github.com/repos/ccao-data/data-architecture"
+)
+GH_API_WORKFLOWS=(
+    "test_dbt_models.yaml"
+    "build_and_test_dbt.yaml"
+)
+GH_API_WORKFLOW_INPUTS=(
+    ""
+    "model.vw_card_res_input model.vw_pin_condo_input"
+)
```

Comment:
This is pretty hideous but works fine. Just iterates through the index of the arrays.

Reply:
**[Nitpick, non-blocking]** I love representing data in shell scripts 🙈 Perhaps a brief comment explaining the data structure would help with readability?

Reply:
Done in [3bee4a2](https://github.com/ccao-data/service-sqoop-iasworld/pull/7/commits/3bee4a2aaaef41999979d57f9dcf0ec5212f2170)!